raspiraw.c

```diff
@@ -636,6 +755,35 @@ static int parse_cmdline(int argc, const char **argv, RASPIRAW_PARAMS_T *cfg)
 	return 0;
 }
 
+int reg_index(struct mode_def *sensor_mode, int r)
```

Comment:
reg_index is no longer used and should be removed.
(Ideally it should have been flagged as static previously so that it only existed within this module rather than globally, at which point the compiler whould have complained that it was defined but not used).